blocks/article-header/article-header.js

```diff
@@ -0,0 +1,73 @@
+import { getMetadata } from '../../scripts/aem.js';
+import { prepURL } from '../../blocks-helpers/prepURL.js';
+
+export default async function decorate(block) {
+  // parse data into an object
+  const articleHeaderData = {
+    title: block.children[0]?.children[0]?.children[0]?.textContent?.trim(),
```

Comment:
```suggestion
    title: block.children?.[0]?.children?.[0]?.children.?.[0]?.textContent?.trim(),
```
Do you think it's worth expanding the optional chaining a bit here? This would additionally help handle the case of `children` being undefined or not an array.

Reply:
yes!